### DISCUSSION TITLE
Download speed

### DISCUSSION THREAD

**Discussion by bezuhv** [aclap-dev/video-downloadhelper#686] [Help and Q&A]
### Your question

How can I set the download speed limit in the new version?!

### Addon & CoApp version and details

version: 9.0.1.22
target: google
channel: stable
lang: ru
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\Виталий\\Desktop",
    "concurrent_downloads_max": 2,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_wide_ui": false,
    "use_sidebar": false,
    "open_count_store": 3,
    "successfull_dl": 4
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I don't quite understand what you're looking for.  If you are talking about controlling how fast VDH downloads content from the Internet, it is my understanding that there is no way to tell VDH to use less than the bandwidth available in your Internet connection.  VDH uses whatever is there.  If you want to limit that, I don't know how to do that.

If that's not your question, do please clarify.
